packages/design-system/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@repo/design-system",
+  "version": "0.0.0",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "types": "./index.ts"
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "type": "module",
+  "license": "MIT",
+  "scripts": {
+    "build": "run-p build:*",
```

Comment:
Until you have other commands that need to run as part of the build, which is unlikely, suggest just removing this and renaming the `build:lib` to `build` instead. Just keep it simple until it needs to change.